README.md

```diff
@@ -20,6 +20,8 @@ urlFragment: azure-search-openai-demo
 > [!IMPORTANT]  
 > As of November 15, 2023, Azure Cognitive Search has been renamed to Azure AI Search.
 
+### Announcing [**JavaScript**](https://aka.ms/azai/js/code), [**.NET**](https://aka.ms/azai/net/code), and [**Java**](https://aka.ms/azai/java/code) samples based on this one in [**Python**](https://aka.ms/azai/py/code). Learn more at  https://aka.ms/azai. 
```

Comment:
What about something like:

Announcing new reference templates in [**JavaScript**](https://aka.ms/azai/js/code), [**.NET**](https://aka.ms/azai/net/code), and [**Java**](https://aka.ms/azai/java/code) based on this one!

Learn more at [https://aka.ms/azai](https://aka.ms/azai).

Reply:
Let's keep the in Python as most folk don't no from the incoming pages that this is a/the Python one. This is being linked to mainly from the RAG docs and other such, so would prefer to keep it as is if that works?